File: java/src/main/java/org/hackerrank/Staircase.java
package org.hackerrank;

public class Staircase {
    public static void staircase(int n) {
        for (int i = 1; i <= n; i++) {
            String element = "#";
            int emptySpaces = n - i;
            String currentLine = " ".repeat(emptySpaces) + element.repeat(i);
            System.out.println(currentLine);
        }
    }
}

File: js/src/staircase.js
function staircase(n) {
	// Write your code here
    let completeLine = "";
	for (let index = 1; index <= n; index++) {
		const element = "#";
		const emptySpaces = n - index;
		const currentLine = " ".repeat(emptySpaces) + element.repeat(index);
        completeLine += currentLine + "\n";
	}
    return completeLine
}


module.exports = staircase;
